### PR TITLE
Able to deploy locally with parity node in dev mode

### PR DESCRIPTION
**Pull request by DieMyst**

### CODE REVIEW

File: tools/deploy/fabfile.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 # change this if you want to deploy on another hosts
 # TO USE: replace values inside <> with your actual values
```
Comment (alari) [inline tools/deploy/fabfile.py]: This comment is outdated, could you update it please to make it clear how to use this magic `info.keys()` thing?

File: tools/deploy/scripts/compose.sh [hunk 2]
```diff
@@ -2,6 +2,18 @@
 
 set -e
 
+if [ -z "$PROD" ]
+then
+    export NAME='node1'
+    export PORTS='25000:25003'
+    export OWNER_ADDRESS=0x00a329c0648769a73afac7f9381e08fb43dbea72
```
Comment (alari) [inline tools/deploy/scripts/compose.sh]: Why is it hardcoded? You're setting it in `fabfile.py`, and hardcode here, is it ok?

File: tools/deploy/scripts/deploy-contract.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+// Get encoded request for deploying contract
+let encodedAbi = NetworkContract.deploy({
+    data: bytecode
+}).encodeABI();
+
+// Send transaction and print the result
+web3.eth.sendTransaction({ from: acc, data: encodedAbi, gas: '4500000' }, "")
+.on('error', console.error)
+.then(function(newContractInstance){
+    console.log(newContractInstance.contractAddress) // instance with the new contract address
```
Comment (alari) [inline tools/deploy/scripts/deploy-contract.js]: Let's print something useful here.

Comment (DieMyst) [reply tools/deploy/scripts/deploy-contract.js]: it printed as result for the bash script

File: tools/deploy/scripts/compose.sh (hunk 2 above)
Comment (DieMyst) [reply tools/deploy/scripts/compose.sh]: it is for dev mode. Run `compose.sh` directly will run containers locally in dev mode.

Comment (alari) [reply tools/deploy/scripts/compose.sh]: Could you add a comment about it here? About why it's a constant

**Pull request merged**